app/controllers/topic_requests_controller.rb

```diff
@@ -0,0 +1,26 @@
+class TopicRequestsController < ApplicationController
+  def index
+    @topic_requests = TopicRequest.all
+    @topic_request = TopicRequest.new
+  end
+
+  def create
+    topic_request = TopicRequest.new(topic_request_params)
+    authorize topic_request
+
+    flash[:notice] = if topic_request.save
+                       'Topic request was succesfully created.'
+                     else
+                       'Error in creating topic request.'
+                     end
+    redirect_to requests_presentations_path
```

Comment:
If we do a redirect here do we get specific error messages for why it wasn't submitted? Like if the user leaves it blank.